### DISCUSSION TITLE
mousewheel.disable() does not prevent swiping on mobile devices

### DISCUSSION THREAD

**Discussion by kabarovsky** [nolimits4web/swiper#6580] [Q&A]
I have a full screen vertical swiper with **mousewheel** enabled which I then disable in some scenarios via **mySwiper.param.mousewheel.disable()**  but that does not have an effect on mobile devices where I can still swipe through sliders.

Any help and suggestions? 

Thanks!

**Comment by ng-asror** [comment]
```
document.addEventListener('click', () => {
    MySwiper.forEach(item => {
        item.mousewheel.disable();  //or enable()
    });
});
```
If this solution works for you, I hope you’ll subscribe, and I’m here to help with any future issues you may have!
